### PR TITLE
Added unpackaged/packaged WPN TestApp

### PR DESCRIPTION
**Pull request by pmpurifoy**
- A basic TestApp that has dependency of ProjectReunion_DLL
- MSIX packager that allows testing the app as a packaged application

### CODE REVIEW

File: test/TestApps/AppLifecycleTestApp - Copy/pch.h [hunk 1]
```diff
@@ -0,0 +1,17 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```
Comment (danielayala94) [inline test/TestApps/AppLifecycleTestApp - Copy/pch.h:1]: Why do we have an "AppLifecycleTestApp - Copy"?

**Comment by danielayala94** [comment]
I'm not sure if I'm following the idea of this PR.

- I see a **WPNWin32TestApp** which looks to be the console test app
- I see the **MSIXPackager** to bring identity to the app above
- Then, what is the role of **_AppLifecycleTestApp - Copy_**?

**Review by danielayala94 — CHANGES_REQUESTED** [review]
I'm not sure if I'm following the idea of this PR.

- I see a **WPNWin32TestApp** which looks to be the console test app
- I see the **MSIXPackager** to bring identity to the app above
- Then, what is the role of **_AppLifecycleTestApp - Copy_**?

File: test/TestApps/AppLifecycleTestApp - Copy/pch.h (hunk 1 above)
Comment (pmpurifoy) [reply test/TestApps/AppLifecycleTestApp - Copy/pch.h:1]: This shouldn't be in the repo. Apparently cleaning the solution doesn't remove old builds. I based the new app off the AppLifeCycleTestApp which created AppLifecycleTestApp, which I built to make sure it compiled. I need to remove this from the build files.

File: test/CppTest/CppTest.vcxproj [hunk 2]
```diff
@@ -324,9 +324,6 @@
       <Project>{b73ad907-6164-4294-88fb-f3c9c10da1f1}</Project>
       <Private>true</Private>
     </ProjectReference>
-    <ProjectReference Include="..\TestApps\AppLifecycleTestApp\AppLifecycleTestApp.vcxproj">
```
Comment (danielayala94) [inline test/CppTest/CppTest.vcxproj:327]: What is the purpose of this change?

File: test/TestApps/AppLifecycleTestApp/AppLifecycleTestApp.vcxproj [hunk 3]
```diff
@@ -42,7 +42,7 @@
     <RootNamespace>AppLifecycleTestApp</RootNamespace>
     <WindowsTargetPlatformVersion>10.0</WindowsTargetPlatformVersion>
     <ProjectSubType>NativeUnitTestProject</ProjectSubType>
-    <ProjectName>AppLifecycleTestApp</ProjectName>
+    <ProjectName>WPNWin32MSIX</ProjectName>
```
Comment (danielayala94) [inline test/TestApps/AppLifecycleTestApp/AppLifecycleTestApp.vcxproj:45]: Why replacing? Doesn't AppLifecycle have its own test app? Seems like are removing the reference in favor of our test app, which is not good.

File: ProjectReunion.sln [hunk 4]
```diff
@@ -31,14 +31,18 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "TestApps", "TestApps", "{AC
 EndProject
 Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "inc", "test\inc\inc.vcxitems", "{B567FE2E-3A03-48D0-B2B5-760CDEC35891}"
 EndProject
-Project("{8BC9CEB8-8B4A-11D0-8D11-00A0C91BC942}") = "AppLifecycleTestApp", "test\TestApps\AppLifecycleTestApp\AppLifecycleTestApp.vcxproj", "{7C502995-59C3-483B-86BA-815985353633}"
```
Comment (danielayala94) [inline ProjectReunion.sln:34]: Why removing the project reference to the AppLifecycle test app?

**Review by danielayala94 — CHANGES_REQUESTED** [review]

File: test/TestApps/AppLifecycleTestApp/AppLifecycleTestApp.vcxproj (hunk 3 above)
Comment (pmpurifoy) [reply test/TestApps/AppLifecycleTestApp/AppLifecycleTestApp.vcxproj:45]: I don't know how it got removed but I am putting it back in the repo.

**Review by danielayala94 — APPROVED** [review]
